forc/src/cli/shared.rs

```diff
@@ -337,3 +347,44 @@ impl From<&Vec<String>> for PrintIrCliOpt {
         Self(print_ir)
     }
 }
+pub struct VerifyIrCliOpt(pub VerifyIr);
```

Comment:
Since these options are completely the same as `PrintIrCliOpt` and this is quite some of copy paste, suggestion would be to unify them and have only, e.g., `IrCliOpt` which will be used for both `print_ir` and `verify_ir` CLI params.